Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -115,6 +115,14 @@ public final class ManifestLoader {
         let toml = try TOMLItem.parse(tomlString)
         let package = PackageDescription.Package.fromTOML(toml, baseURL: baseURL)
         let products = PackageDescription.Product.fromTOML(toml)
+        let errors = parseErrors(toml)
+
+        guard errors.isEmpty else {
+            // FIXME: Output with proper diagnostics infrastucture when available.
+            print("Found errors while parsing manifest of package \(package.name):")
+            errors.forEach { print($0) }
```

Comment:
Shouldn't we bundle these errors into the Error we throw, and let it or its handler be responsible for showing them?